.gitignore

```diff
@@ -0,0 +1 @@
+iterationData.txt
```

Comment:
`.gitignore` can be used to tell Git (and editors) to not show the file when tracking repository changes, which is useful in our case.

Reply:
ah yes,forgot about that as i was pretty tired last night,thanks a bunch

Reply:
also thanks alot for fixing the grammer.